.github/workflows/basic.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         build_type: [Debug, Release]
         compiler: [{c: cl, cxx: cl}]
         pool_tracking: ['ON', 'OFF']
-        shared_library: ['OFF']
+        shared_library: ['ON', 'OFF']
         level_zero_provider: ['ON']
         include:
```

Comment:
you have to add shared_library into all `include`s

Reply:
Added to all Windows builds. The rest is out of scope of this PR.

Reply:
heh, this one is actually all we needed